src/CheckAnswer.js

```diff
@@ -0,0 +1,59 @@
+import { useState} from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowRightToBracket } from '@fortawesome/free-solid-svg-icons';
+
+
+
+function CheckAnswer(props) {
+
+
+    const [userValue, setUserValue] = useState('');
+
+    const [showAnswer, setShowAnswer] = useState(false)
+
+    const scoreKeeper = function(usersResponse) {
+        props.trackScore(usersResponse)
+    }
+
+    //track 2 events within this component, the user submitted answer and the form submit
+
+    const handleInput = function (e) {
+        setUserValue(e.target.value);
+    }
+
+    const handleUserAnswer = function (e) {
+        props.handleSubmit(e, userValue);
+        console.log(userValue);
+
+    }
+
+    
+    async function answerCheck() {
+        props.jAnswer == userValue ?
+        
+        scoreKeeper(props.jScore)
+        : console.log();
+        
+    }
+
+    console.log(props.jAnswer);
+
+    return (
+        <div className="formContainer">
+            <form onSubmit={(e) => { handleUserAnswer(e); answerCheck() }}>
```

Comment:
Try to combine the handle user answer with the answer check together! Probably don't need the async function answerCheck